stdlib/sqlite3/dbapi2.pyi

```diff
@@ -47,12 +48,13 @@ SQLITE_DROP_TEMP_TRIGGER: int
 SQLITE_DROP_TEMP_VIEW: int
 SQLITE_DROP_TRIGGER: int
 SQLITE_DROP_VIEW: int
-SQLITE_DROP_VTABLE: int
-SQLITE_FUNCTION: int
+if sys.version_info[:2] > (3, 6):
```

Comment:
We usually use bare 'sys.version_info` here in typeshed, without the `[:2]` part.

Reply:
Didn't know that! Thanks! 👍